### PR TITLE
RCAL-927: Add ref_file entry for reference pixel subtraction reference file

### PR DESCRIPTION
**Pull request by PaulHuwe**
Resolves [RCAL-927](https://jira.stsci.edu/browse/RCAL-927)

Closes #

This PR adds ref_file entry for reference pixel subtraction reference file.

## Tasks
- [ ] update or add relevant tests
- [ ] update relevant docstrings and / or `docs/` page
- [ ] Does this PR change any API used downstream? (if not, label with `no-changelog-entry-needed`)
  - [ ] write news fragment(s) in `changes/`: `echo "changed something" > changes/<PR#>.<changetype>.rst` (see below for change types)
  - [ ] [start a `romancal` regression test](https://github.com/spacetelescope/RegressionTests/actions/workflows/romancal.yml) with this branch installed (`"git+https://github.com/<fork>/roman_datamodels@<branch>"`)

<details><summary>news fragment change types...</summary>

- ``changes/<PR#>.feature.rst``: new feature
- ``changes/<PR#>.bugfix.rst``: fixes an issue
- ``changes/<PR#>.doc.rst``: documentation change
- ``changes/<PR#>.removal.rst``: deprecation or removal of public API
- ``changes/<PR#>.misc.rst``: infrastructure or miscellaneous change
</details

### CODE REVIEW

**Comment by PaulHuwe** [comment]
Regression Tests:
https://github.com/spacetelescope/RegressionTests/actions/runs/11208792665

**Review by braingram — COMMENTED** [review]
Changes look good to me.

Regression tests show only the expected failures from the new `refpix`:
```
   {'dictionary_item_added': ["root['roman']['meta']['ref_file']['refpix']"]}
```
Waiting to approve until https://github.com/spacetelescope/rad/pull/458 is merged and the rad requirement updated to main.

File: pyproject.toml [hunk 1]
```diff
@@ -18,8 +18,9 @@ dependencies = [
     "gwcs >=0.19.0",
     "numpy >=1.22",
     "astropy >=5.3.0",
-    "rad >= 0.21.0",
+    # "rad >= 0.21.0",
     # "rad @ git+https://github.com/spacetelescope/rad.git",
+    "rad @ git+https://github.com/PaulHuwe/rad.git@RAD-176_RefPixRefFile",
```
Comment (PaulHuwe) [inline pyproject.toml:23]:
```suggestion
    "rad @ git+https://github.com/spacetelescope/rad.git",
```

**Review by braingram — APPROVED** [review]
Thanks for updating the RAD pin.

**Pull request merged**